dashboard-client/src/routes/config/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { FunctionalComponent, h } from "preact";
+import { useCallback, useEffect, useState } from "preact/hooks";
+import * as style from "./style.css";
+
+interface Props { }
+
+const Config: FunctionalComponent<Props> = (props: Props) => {
```

Comment:
IDK about you, but it kind of drives me crazy for every component to be defined and exported from `index.(js|ts|jsx|tsx)`.
How about we just use `index` files for importing and exporting things?

Reply:
I've preferred
```
componentName/
    componentName.tsx
    index.js
    styled.js (/ style.css)
```

And then index.js just looks like
```
export { ComponentName } from './componentName';
```

Because it makes navigation via ctrl+p a bit more direct. Imports aren't any longer, and styles are still scoped to the component.

See [Styled-UI](https://github.com/Faithlife/styled-ui) for a working example. The older components there haven't been updated to the new convention though.